deploy/1_1_deploy_time_lock.ts

```diff
@@ -4,12 +4,17 @@ import type { DeployFunction } from "hardhat-deploy/types";
 const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
```

Comment:
We should also delete the deployment file of the timelock contract from the rsk deployments folder, right?

Reply:
yes and also probably rename the timelock deployment script so it has a '12_' prefix, since it's really only deployed right now

Reply:
Yep that's a good idea

Reply:
@AndrzejGisWideLab how will doing that work on a new chain? Does it mean we'll deploy everything using a regular wallet deployer and only then move everything to the timelock?

Reply:
@kazazor I think it's too complex to talko here in comments. Let's talk it through live